homeassistant/components/anthropic/config_flow.py

```diff
@@ -128,21 +132,50 @@ async def async_step_init(
     ) -> ConfigFlowResult:
         """Manage the options."""
         options: dict[str, Any] | MappingProxyType[str, Any] = self.config_entry.options
+        errors: dict[str, str] = {}
 
         if user_input is not None:
             if user_input[CONF_RECOMMENDED] == self.last_rendered_recommended:
                 if user_input[CONF_LLM_HASS_API] == "none":
                     user_input.pop(CONF_LLM_HASS_API)
-                return self.async_create_entry(title="", data=user_input)
 
-            # Re-render the options again, now with the recommended options shown/hidden
-            self.last_rendered_recommended = user_input[CONF_RECOMMENDED]
+                if (
+                    user_input.get(
+                        CONF_THINKING_BUDGET_TOKENS, RECOMMENDED_THINKING_BUDGET_TOKENS
+                    )
+                    >= MIN_THINKING_BUDGET
+                    and user_input.get(CONF_CHAT_MODEL, RECOMMENDED_CHAT_MODEL)
+                    not in THINKING_MODELS
```

Comment:
Is this something we can check in their API?

Reply:
No, the API only returns release date and human-readable name